wgpu-types/src/lib.rs

```diff
@@ -601,17 +781,25 @@ impl Default for ColorWrite {
     }
 }
 
+/// Describes the depth/stencil state in a render pipeline.
 #[repr(C)]
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 pub struct DepthStencilStateDescriptor {
+    /// Format of the depth/stencil buffer, must be special depth format.
     pub format: TextureFormat,
+    /// If disabled, depth will not be written to.
     pub depth_write_enabled: bool,
+    /// Comparison function used to compare depth values in the depth test.
     pub depth_compare: CompareFunction,
+    /// Stencil state used for front faces.
     pub stencil_front: StencilStateFaceDescriptor,
+    /// Stencil state used for back faces.
     pub stencil_back: StencilStateFaceDescriptor,
+    /// Stencil values are AND'd with this mask when reading from the stencil buffer. Only low 8 bits are used.
```

Comment:
Also, when written... which is confusing. We should probably rename it to just `stencil_mask`

Reply:
Wat. Yeah that would be helpful :P